pkg/plugin/v2/webhook.go

```diff
@@ -93,18 +95,18 @@ func (p *createWebhookPlugin) Validate(_ *config.Config) error {
 	return nil
 }
 
-func (p *createWebhookPlugin) GetScaffolder(c *config.Config) (scaffold.Scaffolder, error) { // nolint:unparam
+func (p *createWebhookPlugin) GetScaffolder() (scaffold.Scaffolder, error) {
 	// Load the boilerplate
 	bp, err := ioutil.ReadFile(filepath.Join("hack", "boilerplate.go.txt")) // nolint:gosec
 	if err != nil {
 		return nil, fmt.Errorf("unable to load boilerplate: %v", err)
 	}
 
 	// Create the actual resource from the resource options
-	res := p.resource.NewResource(&c.Config, false)
-	return scaffold.NewV2WebhookScaffolder(&c.Config, string(bp), res, p.defaulting, p.validation, p.conversion), nil
+	res := p.resource.NewResource(p.config, false)
+	return scaffold.NewV2WebhookScaffolder(p.config, string(bp), res, p.defaulting, p.validation, p.conversion), nil
```

Comment:
Same here. The `NewV2WebhookScaffolder` has the required data, so could not the `p.resource.NewResource(p.config, false)` be inside of?